src/ImageKit/Resource/GuzzleHttpWrapper.php

```diff
@@ -161,6 +166,10 @@ protected function handleException(Exception $e)
         $headers = [];
         $body = $e->getMessage();
 
+        if($status==0){
+            $status=100;
```

Comment:
Why are we doing this? `statusCode:100` has a particular meaning.
```
The HTTP 100 Continue informational status response code indicates that everything so far is OK and that the client should continue with the request or ignore it if it is already finished.
```

Reply:
Fixed